source/main/utils/buildAppMenus.js

```diff
@@ -92,13 +94,44 @@ export const buildAppMenus = async (
     item.checked = isBlankScreenFixActive;
   };
 
+  const setRtsFlags = async (enable: boolean): Promise<void> => {
+    const translation = getTranslation(translations, 'menu');
+    const rtsFlagsDialogOptions = {
+      buttons: [
+        translation('helpSupport.rtsFlagsDialogConfirm'),
+        translation('helpSupport.rtsFlagsDialogCancel'),
+      ],
+      type: 'warning',
+      title: enable
+        ? translation('helpSupport.enableRtsFlagsDialogTitle')
+        : translation('helpSupport.disableRtsFlagsDialogTitle'),
+      message: enable
+        ? translation('helpSupport.enableRtsFlagsDialogMessage')
+        : translation('helpSupport.disableRtsFlagsDialogMessage'),
+      defaultId: 1,
+      cancelId: 1,
+      noLink: true,
+    };
+
+    const { response } = await dialog.showMessageBox(
+      mainWindow,
+      rtsFlagsDialogOptions
+    );
+    if (response === 0) {
+      enable
+        ? setRtsFlagsAndRestart(environment.network, RTS_FLAGS)
+        : setRtsFlagsAndRestart(environment.network, []);
```

Comment:
I don't want to be a nitpicker but personally I dislike this notation. It's looking like an expression that should be assigned to some variable or a return value.

Please consider following:

```js
setRtsFlagsAndRestart(environment.network, enable ? RTS_FLAGS : []);
```

Reply:
it has been changed to an if. Thanks @marcin-mazurek